assets/src/application/ui/display/EntityListLegend/types.ts

```diff
@@ -0,0 +1,12 @@
+import { Icon } from '@appDisplay/espressoIcon';
+
+type Icons = {
+	[key in Icon]?: string;
+};
```

Comment:
My suggestion here would be to create an array of objects with descriptive keys like this:
```ts
type IconDescriptions = Array<{
    icon: Icon;
    description: string;
}>
```

Reply:
agree with more explicit approach

Reply:
updated